test/backend-test/test-rabbitmq.js

```diff
@@ -0,0 +1,53 @@
+const { describe, test } = require("node:test");
+const assert = require("node:assert");
+const { RabbitMQContainer } = require("@testcontainers/rabbitmq");
+const { RabbitMqMonitorType } = require("../../server/monitor-types/rabbitmq");
+const { UP, DOWN, PENDING } = require("../../src/util");
+
+describe("RabbitMQ Single Node", {
+    skip: !!process.env.CI && (process.platform !== "linux" || process.arch !== "x64"),
```

Comment:
It seems that this test blocked local testing on my Windows machine. It is also not able to run on a self-hosted X64 debian actions runner. What is the prerequisite to run this actually?

Reply:
This is the platform support for running docker containers in GHA without additional steps.

Reply:
> What is the prerequisite to run this actually?

It assumes that docker is running on the system. AFAIK by default it searches for a unix socket at `unix:///var/run/docker.sock` in linux or a named pipe at `npipe:////./pipe/docker_engine` in windows.
On the debian machine, is the test skipped or does it throw an error?

Reply:
> Could not find a working container runtime strategy.

Full log:
https://github.com/louislam/uptime-kuma/actions/runs/11531916870/job/32103162510

Actually, I just realize the mqtt test has the same issue.

> It assumes that docker is running on the system.
My actions runner on Debian indeed don't have Docker. Maybe that is the reason.